app/helpers/reference_helper.rb

```diff
@@ -0,0 +1,38 @@
+module ReferenceHelper
+
+  def after_desired_date?
+    Time.zone.today > Date.parse('2015-12-31')
```

Comment:
Can we put this to `settings.yml` and read it from the `Settings` class? Maybe the `settings.yml` should even read the value from `ENV` so that we don't have to do a full release if this changes.